vms/avm/network/network.go

```diff
@@ -116,71 +204,59 @@ func (n *network) AppGossip(ctx context.Context, nodeID ids.NodeID, msgBytes []b
 		return nil
 	}
 
-	if err := n.issueTx(tx); err == nil {
+	err = n.mempool.Add(tx)
+	if err == nil {
 		txID := tx.ID()
+		n.txPushGossiper.Add(tx)
+		if err := n.txPushGossiper.Gossip(ctx); err != nil {
```

Comment:
Should we try to better batch these gossip invocations to reduce the number of messages we push (no more than one every x ms)? Or is that handled in the SDK?

Reply:
AFAICT we would be safe even if we didn't manually invoke this: https://github.com/ava-labs/avalanchego/blob/ee10054e2c7af3c73880bce4e795ebb85f3bf668/network/p2p/gossip/gossip.go#L331-L347

Reply:
Yeah - this is kind of an independent suggestion to this PR (as this is the same behavior as before)... But we could play around with batching